seshdash/utils/alert.py

```diff
@@ -13,18 +14,27 @@
 # Alert_Rule.objects.create(site = site, check_field="battery_voltage", value=25, operator="lt")
 
 # Rules are based on sites. So you need to define a rule for each site if you want to check same configurations in several sites.
-# A Sesh_Alert object is created for each alert triggered and an email is send if the rule has send_mail option true
+# A Sesh_Alert object is created for each 'alert triggered and an email is send if the rule has send_mail option true
 def alert_check(data_point):
     ops = {'lt': lambda x,y: x<y,
            'gt': lambda x,y: x>y,
            'eq' : lambda x,y: x==y,
     }
-    recipients = []
+    email_recipients = []
+    sms_recipients = []
     content = {}
 
     rules = Alert_Rule.objects.filter(site = data_point.site)
     for rule in rules:
-        real_value = getattr(data_point,rule.check_field)
+
```

Comment:
I need some comments in this section as what we're doing is a bit complex. We might forget what's going on later.